packages/builder/src/components/userInterface/AppPreview/iframeTemplate.js

```diff
@@ -35,12 +21,45 @@ export default ({
         }
     </style>
     <script>
-        window["##BUDIBASE_FRONTEND_DEFINITION##"] = ${frontendDefinition};
+      function receiveMessage(event) { 
```

Comment:
NAB: You can avoid adding this function as a string by doing something like this in the parent svelte component:

```js
iframe.contentWindow.receiveMessage = function() {
....
}
```